src/main/java/frc/robot/subsystems/StorageSubsystem.java

```diff
@@ -22,23 +22,25 @@
   public StorageSubsystem(MoPDP pdp) {
     storage = new SafeSP(Constants.STORAGE_VICTORSP_PWM_CHAN, SAFE_SPEED, SAFE_COOLDOWN_MS, pdp
         .MakeOvercurrentMonitor(Constants.STORAGE_VICTORSP_PDP_CHAN, UNSAFE_CURRENT_LIMIT, UNSAFE_CURRENT_TIMEOUT_MS));
-    stop();
+    storage.setInverted(false);
   }
 
   public void run() {
     speed = MoPrefs.getStorageSpeed();
+    storage.set(speed);
   }
 
   public void reverse() {
-    speed = -MoPrefs.getStorageSpeed();
+    speed = -1 * MoPrefs.getStorageSpeed();
```

Comment:
Why -1* instead of just -?

Reply:
Personally, I find it more legible. The `-` can be easy to miss.